code/modules/antagonists/heretic/knowledge/side_rust_cosmos.dm

```diff
@@ -18,6 +18,26 @@
 	cost = 1
 	route = PATH_SIDE
 
+/datum/heretic_knowledge/entropy_pulse
+	name = "Pulse of Entropy"
+	desc = "Allows you to transmute 20 irons and 2 garbage items to fill the surrounding vicinity of the rune with rust."
+	gain_text = "Reality begins to whisper to me. To give it its entropic end."
+	required_atoms = list(
+		/obj/item/stack/sheet/iron = 20,
+		/obj/item/trash = 2
+	)
+	cost = 1
```

Comment:
Costs, costs, costs. Does this need you to risk everything to sac someone or suck up one influence ball? Is that really necessary? This is the equivalent of making syndicates pay for secondary objective items. I'd say to give this for free honestly, but that is up to @MrMelbert to decide

Reply:
I'd rather we finally implement some more alternatives for obtaining knowledge, would make it generally easier to add more side paths

(Cool ones for rust might be "cover x tiles in rust" or "cover all of x room in rust" or "rust hand a silicon and bring them to your rune", whatever idk)